lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -467,6 +467,8 @@ def _verify_dl_flow(self, dl_flow_rules=None):
                     },
                 },
             )
+            print ("Created ", len(total_dl_ovs_flows_created),
```

Comment:
should we describe on the log that these are for number of OVS flows?

Reply:
Addressed in latest version